examples/customid/domain/id.go

```diff
@@ -0,0 +1,24 @@
+package domain
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+// ID64 implements ID interface
+type ID64 int64

```

Comment:
Nice example! Should it be located in a core package called something like `/ids/int64`?